src/types/commands.ts

```diff
@@ -501,6 +501,12 @@ export interface PasteCommand {
   pasteOption?: ClipboardPasteOptions;
 }
 
+export interface RepeatPasteCommand {
+  type: "REPEAT_PASTE";
```

Comment:
there are two different strategies for the two repeatable commands from state-full plugins (STOP_EDITION and PASTE)
PASTE => new command + new state in the plugin
STOP_EDITION => check core command

Couldn't paste work similarly to STOP_EDITION ?

Reply:
No. I tried to have a behaviour like that at first, but I couldn't do it correctly. It works for stop edition, because the behaviour is simple, but the core commands dispatched by paste depends on where we paste/the state of the spreadsheet. Some exmples : 

- paste at the edge of the sheet. We will add new col/rows to not paste something out of the sheet. But now if we repeat the command, we don't want to add new col rows.
    - the inserse is also true. If we repeat at the edge of the sheet, we want to add new col/rows.
- similar problem with filters (maybe with merge too?)
-  when pasting CF, I have no idea looking at the `ADD_CONDITIONAL_FORMAT` what ranges were added by paste, and what ranges were there before

Reply:
ok thanks for explaining. Could you add the explanation in the commit message so that others reading the commit will understand our choices. People won't read the 50+ comments on the PR